Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -290,9 +231,9 @@ void HoI4::World::addNeutrality()
 {
 	for (auto country: countries)
 	{
-		if (majorIdeologies.count(country.second->getGovernmentIdeology()) == 0)
+		if (ideologies->getMajorIdeologies().count(country.second->getGovernmentIdeology()) == 0)
```

Comment:
Uh, if you're copying countries and then setting their ideologies, does this actually get saved?
Same below in the other instances.

Reply:
It's copying shared pointers to countries, so it works.

It's not good, and will be addressed when cleanup gets this far, but it works.